### PR TITLE
Add docker-compose for schema-registry

### PR DESCRIPTION
**Pull request by sailxjx**

### CODE REVIEW

**Comment by ewencp** [comment]
ok to test

**Review by sgerrand — APPROVED** [review]
Works for me. 👍

File: examples/schema-registry/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    network_mode: host
+    depends_on:
+      - zookeeper
+      - kafka
+    environment:
+      SCHEMA_REGISTRY_KAFKASTORE_CONNECTION_URL: localhost:32181
+      SCHEMA_REGISTRY_HOST_NAME: schema-registry
+      SCHEMA_REGISTRY_LISTENERS: http://0.0.0.0:38081
+
+  kafka-rest:
```
Comment (mageshn) [inline examples/schema-registry/docker-compose.yml]: Just wondering why Rest Proxy is needed in schema registry compose. If you need Rest proxy, i would probably change the folder to kafka-rest

Comment (sailxjx) [reply examples/schema-registry/docker-compose.yml]: I just need an out of box compose, for testing or developing

**Pull request closed without merging**